src/App.tsx

```diff
@@ -19,6 +19,8 @@ export default function App() {
 	const [playing, setPlaying] = useState(false);
 	const [frameStart] = useState();
 	const [frameEnd, setEnd] = useState(500);
+
+	const [value, setValue] = useState(0);
```

Comment:
what is this?

Reply:
I was using this before but ended up not needing it. Removed now.